docs/source/user-guide/sql/dml.md

```diff
@@ -0,0 +1,60 @@
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+
+# DML
+
+## COPY
+
+Copy a table to file(s). Supported file formats are `parquet`, `csv`, and `json`.
+
+<pre>
+COPY <i><b>table_name</i></b> TO '<i><b>file_name</i></b>' [ ( <i><b>option</i></b> [, ... ] ) ]
+
+where <i><b>option</i></b> can be one of:
+    FORMAT <i><b>format_name</i></b>
+    PER_THREAD_OUTPUT <i><b>boolean</i></b>
+    ROW_GROUP_SIZE <i><b>integer</i></b>
+    ROW_GROUP_LIMIT_BYTES <i><b>integer</i></b>
+</pre>
+
+```sql
+> COPY source_table TO 'table' (FORMAT parquet, PER_THREAD_OUTPUT true);
```

Comment:
```suggestion
> COPY source_table TO 'filename.parquet' (FORMAT PARQUET, PER_THREAD_OUTPUT TRUE);
```

Reply:
I added [an explanation and another example](https://github.com/apache/arrow-datafusion/pull/7362/commits/487ffbf4d6e9e39990e1d187ac7605441a743fc7) to clarify when one should specify a file name or a directory name.